src/main/java/com/spotify/docker/client/DefaultDockerClient.java

```diff
@@ -2599,6 +2599,38 @@ public VolumeList listVolumes(ListVolumesParam... params)
     return request(GET, SECRET_LIST, resource, resource.request(APPLICATION_JSON_TYPE));
   }
 
+  @Override
+  public List<Secret> listSecrets(final Secret.Criteria criteria)
```

Comment:
Does the API accept more than one filter at a time? If so, maybe this should take var args Criteria.

Reply:
The API takes only one filter, just like for Config.